sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -152,7 +152,7 @@
 	public static final String DATABASE_NAME = "sormas.db";
 	// any time you make changes to your database objects, you may have to increase the database version
 
-	public static final int DATABASE_VERSION = 247;
+	public static final int DATABASE_VERSION = 249;
```

Comment:
Here the version is 249, but in the case below, it is 248, is this correct? @FredrikSchaeferVitagroup

Reply:
Hmm... Okay, I remembered this to be one larger. Alright, I'm gonna fix this.

Reply:
DATABASE_VERSION should be incremented, otherwise it will not run your latest migration.